nemo_curator/classifiers/domain.py

```diff
@@ -28,19 +28,31 @@
 from nemo_curator.datasets import DocumentDataset
 
 DOMAIN_IDENTIFIER = "nvidia/domain-classifier"
+DOMAIN_BASE_MODEL = "microsoft/deberta-v3-base"
+MULTILINGUAL_DOMAIN_IDENTIFIER = "nvidia/multilingual-domain-classifier"
+MULTILINGUAL_DOMAIN_BASE_MODEL = "microsoft/mdeberta-v3-base"
 
 
 @dataclass
 class DomainModelConfig:
-    model: str = "microsoft/deberta-v3-base"
+    identifier: str = DOMAIN_IDENTIFIER
+    base_model: str = DOMAIN_BASE_MODEL
+    fc_dropout: float = 0.2
+    max_len: int = 512
+
+
+@dataclass
+class MultilingualDomainModelConfig:
+    identifier: str = MULTILINGUAL_DOMAIN_IDENTIFIER
+    base_model: str = MULTILINGUAL_DOMAIN_BASE_MODEL
```

Comment:
nit : but given `DomainModelConfig` is a dataclass, do we need `MultilingualDomainModelConfig`? Since in it's current state `MultilingualDomainModelConfig` is just `DomainModelConfig` with non-default values

Reply:
Updated, thanks!